pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -20,6 +20,7 @@ import (
 	"errors"
 	"fmt"
 
+	scmeta "github.com/kubernetes-incubator/service-catalog/pkg/api/meta"
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/server"
 	"github.com/kubernetes-incubator/service-catalog/pkg/storage/tpr"
```

Comment:
no. the functionality that is in `scmeta` (which TPR functionality uses, but isn't TPR related) was moved from elsewhere, so this `import` had to be added, along with changes below